Dockerfile

```diff
@@ -1,41 +1,41 @@
-# WARNING: keep this file in sync with taskcluster/docker/balrog-backend/Dockerfile
+# WARNING: keep this file in sync with agent/Dockerfile
+ARG PYTHON_VERSION=3.13
+FROM ghcr.io/astral-sh/uv:python${PYTHON_VERSION}-bookworm-slim AS builder
```

Comment:
Nice to see more multi-stage builds! We should probably document some best practices for our docker images at some point.